content/fr/docs/tutorials/hello-minikube.md

```diff
@@ -16,7 +16,7 @@ card:
 
 {{% capture overview %}}
 
-Ce tutoriel vous montre comment exécuter une simple application Hello World Node.js sur Kubernetes en utilisant [Minikube](/docs/getting-start-guides/minikube) et Katacoda.
+Ce tutoriel vous montre comment exécuter une simple application Hello World Node.js sur Kubernetes en utilisant [Minikube](/docs/getting-started-guides/minikube) et Katacoda.
```

Comment:
```suggestion
Ce tutoriel vous montre comment exécuter une simple application Hello World Node.js sur Kubernetes en utilisant [Minikube](/docs/getting-started-guides/minikube/) et Katacoda.
```

Reply:
IMHO, no need, as such links will be handled by server.